lib/carbon/writer.py

```diff
@@ -127,7 +127,7 @@ def writeCachedDataPoints():
         try:
             state.database.create(metric, archiveConfig, xFilesFactor, aggregationMethod)
             instrumentation.increment('creates')
-        except Exception:
+        except Exception, e:
```

Comment:
are we using python2.7? maybe better to use the "as" exception handling in the case that we somehow arrive at python3.
https://stackoverflow.com/questions/2535760/python-try-except-comma-vs-as-in-except

Reply:
yes using python2.7 
I will update this later to python3 
The actual carbon (not the forked version) is py3 compatible. but our forked version isnt.